console/src/tasks.rs

```diff
@@ -182,6 +230,19 @@ impl State {
         let rows = self.sorted_tasks.iter().filter_map(|task| {
             let task = task.upgrade()?;
             let task = task.borrow();
+
+            let fields = Spans::from(task.fields().iter().fold(Vec::default(), |mut acc, f| {
+                acc.extend(vec![
+                    Span::styled(
+                        f.name.clone(),
+                        Style::default().add_modifier(Modifier::BOLD),
+                    ),
+                    Span::from("="),
+                    Span::from(format!("{} ", f.value)),
+                ]);
+                acc
+            }));
```

Comment:
hmm, this change is actually a performance hit, because we are now formatting the fields again *every* time we update the view, rather than just once.

it might be worth having the `Task` struct store the formatted representation as well as the structured representation (which we would use for things like sorting/filtering by fields). we could have some logic for determining when to re-format the fields if the task span records new fields (which currently won't happen, but we may add in the future).

again, we don't _have_ to do that in this PR, but it might be nice to do in the future as an optimization?